keps/sig-network/1111-dynamically-re-configure-lb-node-set/README.md

```diff
@@ -0,0 +1,681 @@
+
+#### Beta
+
+- Documentation is changed for ETP=cluster services to highlight the change with
+  regards to `healthCheckNodePort`.
+- Testing of Kubernetes with the feature flag enabled is green and reports no
+  major incidents.
+
+#### GA
+
+- Version skew delta has passed.
+
+### Upgrade / Downgrade Strategy
+
+<!--
+If applicable, how will the component be upgraded and downgraded? Make sure
+this is in the test plan.
+
+Consider the following in developing an upgrade/downgrade strategy for this
+enhancement:
+- What changes (in invocations, configurations, API use, etc.) is an existing
+  cluster required to make on upgrade, in order to maintain previous behavior?
+- What changes (in invocations, configurations, API use, etc.) is an existing
+  cluster required to make on upgrade, in order to make use of the enhancement?
+-->
+
+### Version Skew Strategy
+
+The possible version skew between the control and data plane in a Kubernetes
+cluster and the fact that this proposal depends on having the data plane
+implement an HC which the control plane can target, makes the implementation
+plan tricky. In a world where this KEP has been agreed upon, we would need to
+begin by implementing the HC in the data plane, allow it to soak until the
+version drift has reached its maximum supported delta, and then enable and start
+targeting the HC from the control plane.
+
+## Production Readiness Review Questionnaire
+
+<!--
+
+Production readiness reviews are intended to ensure that features merging into
+Kubernetes are observable, scalable and supportable; can be safely operated in
+production environments, and can be disabled or rolled back in the event they
+cause increased failures in production. See more in the PRR KEP at
+https://git.k8s.io/enhancements/keps/sig-architecture/1194-prod-readiness.
+
+The production readiness review questionnaire must be completed and approved
+for the KEP to move to `implementable` status and be included in the release.
+
+In some cases, the questions below should also have answers in `kep.yaml`. This
+is to enable automation to verify the presence of the review, and to reduce review
+burden and latency.
+
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+-->
+
+### Feature Enablement and Rollback
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [X] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: LoadBalancerHealthCheckNodePort?...I suck at assigning
+    names.
+  - Components depending on the feature gate: KCCM
+
+###### Does enabling the feature change any default behavior?
+
+Yes, we stop changing the load balancer backend pool depending on node state.
```

Comment:
who polls the service proxy's healthz status? the LB implementation?

Reply:
Yes

Reply:
can we make it explicit in the text?